src/test/java/uk/gov/hmcts/ccd/ElasticsearchIT.java

```diff
@@ -0,0 +1,1447 @@
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.LAST_MODIFIED_DATE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.LAST_STATE_MODIFIED_DATE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.MULTI_SELECT_LIST_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.NESTED_COLLECTION_TEXT_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.NESTED_NUMBER_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.NESTED_NUMBER_FIELD_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.NUMBER_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.NUMBER_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.PARTIAL_PHONE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.PHONE_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.PHONE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.PHONE_VALUE_WITH_SPACE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.POST_CODE_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.POST_CODE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.STATE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.STATE_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.STREET_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TEXT_ALIAS;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TEXT_AREA_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TEXT_AREA_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TEXT_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TEXT_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TOWN_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.TOWN_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.VALUE_SUFFIX;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.YES_OR_NO_FIELD;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.YES_OR_NO_VALUE;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.alias;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.caseData;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.caseTypesParam;
+import static uk.gov.hmcts.ccd.test.ElasticsearchTestHelper.createPostRequest;
+
+@RunWith(Enclosed.class)
+public class ElasticsearchIT extends ElasticsearchBaseTest {
+
+    private static final String DATA_DIR = "elasticsearch/data";
+
+    @Inject
+    private WebApplicationContext wac;
+
+    @Mock
+    private Authentication authentication;
+
+    @Mock
+    private SecurityContext securityContext;
+
+    @Inject
+    private ApplicationParams applicationParams;
+
+    private MockMvc mockMvc;
+
+    @BeforeAll
+    public static void initElastic(@Autowired EmbeddedElastic embeddedElastic) throws IOException,
```

Comment:
is it worth putting a LOG message here so we can track elastic search start (and stop)?

Reply:
Will add a ES start message at this level